src/app.py

```diff
@@ -88,7 +123,9 @@ def process_object_factory(file_builder, object_factory):
 
 
 def instantiate_file_builder(factory_definition,
-                             dev_file_builder_args):
+                             dev_file_builder_args,
```

Comment:
Would it be cleaner to call "get_google_drive_connector" in here, given we have access to factory_definitions? 

Some untidiness would be introduced from the args.g_drive_root, but alternatively this flag could be moved to the "shared arguments" of the config?